NEWS.md

```diff
@@ -56,6 +56,7 @@ New library functions
 * `include` now accepts an optional `mapexpr` first argument to transform the parsed
   expressions before they are evaluated ([#34595]).
 * New function `bitreverse` for reversing the order of bits in a fixed-width integer ([#34791]).
+* New `bitrotate(x, k)` function implements bitwise rotation ([#33937]).
```

Comment:
```suggestion
* New function `bitrotate(x, k)` for rotating the buts in a fixed-width integer ([#33937]).
```

Reply:
Sorry, I didn't notice that was changed; I just wanted to rebase to keep the history linear.

Reply:
Except I suspect you meant "for rotating the bits".